CHANGES.rst

```diff
@@ -4,8 +4,8 @@ Changelog of z3c.dependencychecker
 2.5 (unreleased)
 ----------------
 
-- Nothing changed yet.
-
+- Check in every top level folder if the .egg-info folder is in them.
```

Comment:
I think there's a phrase at the top of `USAGE.rst` that talks about "egg-info in the root or in src/", that one also has to be adjusted.

Reply:
@gforcada, I've fixed up the USAGE.rst documentation a bit in https://github.com/reinout/z3c.dependencychecker/pull/89/commits/938c4e6f21a380e47c99a6ae3f67a1aeb9d11e54 (I pushed it to your branch).

Does that look OK?

If so: can *you* do the merge + release? Then we know for sure that I set the right permissions on the pypi project :-)

Reply:
Thanks for the fix!

I will do the release, yes, thanks for granting me rights!